src/Resources/contao/dca/tl_entity_import_source.php

```diff
@@ -81,7 +81,7 @@
 
     // Subpalettes
     'subpalettes' => [
-        'retrievalType_http'               => 'sourceUrl,httpMethod,httpAuth,fileType',
+        'retrievalType_http'               => 'dontCheckSSL,sourceUrl,httpMethod,httpAuth,fileType',
```

Comment:
Could you please move it after the sourceUrl as this should be the first the user thinks of.

Reply:
OK, I will do.